js/webpack/app.js

```diff
@@ -67,7 +67,7 @@ module.exports = {
       },
       {
         test: /\.js$/,
-        include: /node_modules\/(material-chip-input|ethereumjs-tx)/,
```

Comment:
Is this necessary? I think the code should work in any browser supporting ES6. Do we support other browsers anyway?